DangoPlop/Assets/Scripts/Ball_Behavioiur.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Ball_Behavioiur : MonoBehaviour {
+    public GameObject player;
+    public GameObject ball;
+    public Vector2 ballPos;
+    public Rigidbody2D rb;
+    public int thrust;
+    public CircleCollider2D circle;
+    public int counter;
+
+    // Use this for initialization
+    void Start () {
+        ball = GameObject.FindGameObjectWithTag("Ball");
```

Comment:
Why do we need to find the object with the tag? I thought this script is attached to each individual ball prefab?